src/main/java/bobthebot/bob/BobTheBot.java

```diff
@@ -0,0 +1,55 @@
+package bobthebot.bob;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+import bobthebot.exceptions.BobException;
+import bobthebot.tasks.ToDoList;
+import bobthebot.utils.Parser;
+import bobthebot.utils.Storage;
+import bobthebot.utils.Ui;
+
+/**
+ * BobTheBot is a friendly CLI Bot that can help with managing tasks.
+ * */
+
+public class BobTheBot {
+    private Storage storage;
+    private ToDoList list;
+    private Parser parser;
+
+    /**
+     * Constructor for instance of BobTheBot
```

Comment:
```suggestion
     * Constructor for instance of BobTheBot.
```

Should this have punctuation? [relevant coding standard](https://se-education.org/guides/conventions/java/intermediate.html#:~:text=Punctuation%20behind%20each%20parameter%20description)